src/main/osd/osd.h

```diff
@@ -189,6 +189,14 @@ typedef enum {
     OSD_GPS_LAP_TIME_CURRENT,
     OSD_GPS_LAP_TIME_PREVIOUS,
     OSD_GPS_LAP_TIME_BEST3,
+    OSD_CUSTOM_MSG1,
+    OSD_CUSTOM_MSG2,
+    OSD_CUSTOM_MSG3,
+    OSD_CUSTOM_MSG4,
+    OSD_CUSTOM_MSG5,
+    OSD_CUSTOM_MSG6,
+    OSD_CUSTOM_MSG7,
+    OSD_CUSTOM_MSG8,
```

Comment:
```suggestion
#if OSD_CUSTOM_MSG_COUNT > 4
    OSD_CUSTOM_MSG5,
    OSD_CUSTOM_MSG6,
    OSD_CUSTOM_MSG7,
    OSD_CUSTOM_MSG8,
#endif    
```
Or one #if per line, but that is probably overkill ...